bouncerscript/task.py

```diff
@@ -96,9 +96,10 @@ def check_product_names_match_aliases(context):
 
 def check_product_names_match_nightly_locations(context):
     """Double check that nightly products are as expected"""
-    products = context.task["payload"]["bouncer_products"]
-    if sorted(products) != sorted(BOUNCER_PATH_REGEXES_PER_PRODUCT.keys()):
-        raise TaskVerificationError("Products {} don't correspond to nightly ones".format(products))
+    products = set(context.task["payload"]["bouncer_products"])
+    valid_products = set(BOUNCER_PATH_REGEXES_PER_PRODUCT.keys())
+    if not products.issubset(valid_products):
```

Comment:
I'm not sure this is correct. Thing is, whenever we want to update the nightly products, we need to update all of them at once, not a subset of them. So if say, there's an in-tree misconfiguration/malicious configuration and, say, only two of the nightlies products are in the payload, that means we'll end up with various nightly products serving different versions. This should be strict comparison, either all of them or none.

On the other hand, I understand why you approached it this way. MSi is windows specific and hence won't always be there. But we only have one `bouncer-locations` job per nightly release so I don't see why not keep as it was and simply enrich the list of products with `firefox-msi-latest-ssl` as well?